make/sources.js

```diff
@@ -1,16 +1,18 @@
 "use strict";
 
 exports.libcs = [
-    "src/js/libcs/Namespace.js",
-    "src/js/libcs/Accessors.js",
     "src/js/libcs/CSNumber.js",
     "src/js/libcs/List.js",
-    "src/js/libcs/Essentials.js",
     "src/js/libcs/General.js",
+    "src/js/libcs/Essentials.js",
+    "src/js/libcs/Namespace.js",
+    "build/js/Compiled.js",
```

Comment:
That's JavaScript code transpiled from CindyScript. Perhaps I should choose a different name here. Any suggestions?